cmd/healthchecker/options/options.go

```diff
@@ -47,8 +47,8 @@ type HealthCheckerOptions struct {
 func (hco *HealthCheckerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&hco.Component, "component", types.KubeletComponent,
 		"The component to check health for. Supports kubelet, docker and cri")
-	fs.StringVar(&hco.SystemdService, "systemd-service", "",
-		"The underlying systemd service responsible for the component. Set to the corresponding component for docker and kubelet, containerd for cri.")
+	fs.StringVar(&hco.Service, "service", "",
```

Comment:
For backwards compatibility keep systemd-service as the flag name. You can use...
```
if runtime.GOOS == "linux" {
  fs.StringVar(&hco.Service, "systemd-service", "",...)
} else {
  fs.StringVar(&hco.Service, "service", "",
}
```